src/components/networked-avatar.js

```diff
@@ -0,0 +1,6 @@
+AFRAME.registerComponent("networked-avatar", {
```

Comment:
This can probably be the same thing as `player-info`. I made it its own thing because `player-info` runs a routine to set the namatag text and the src of the gltf-model-plus every time its attributes change, and these hand poses change all the time. When they're combined, we'll only do those things if those were the attributes that actually changed.

Reply:
Also could be useful to split these up for reliable vs unreliable channel delivery. I don't know enough about NAF to know if that's the right separation